src/Settings/Builders/Configure.hs

```diff
@@ -9,10 +9,10 @@ configureBuilderArgs = do
     gmpPath    <- expr gmpBuildPath
     libffiPath <- expr libffiBuildPath
     mconcat [ builder (Configure gmpPath) ? do
-                hostPlatform  <- getSetting HostPlatform
                 buildPlatform <- getSetting BuildPlatform
```

Comment:
It would be nice to compute these per stage, so we don't need to do the unintuive renaming below.